cmd/cli/kubectl-kyverno/utils/common/kyverno_policies_types.go

```diff
@@ -204,3 +205,66 @@ OuterLoop:
 
 	return engineResponses, nil
 }
+
+func combineRuleResponses(imageResponse engineapi.EngineResponse) engineapi.EngineResponse {
+	if imageResponse.PolicyResponse.RulesAppliedCount() == 0 {
+		return imageResponse
+	}
+	ruleResponses := imageResponse.PolicyResponse.Rules
+	var failRuleResponses []engineapi.RuleResponse
+	var errorRuleResponses []engineapi.RuleResponse
+	var passRuleResponses []engineapi.RuleResponse
+	var skipRuleResponses []engineapi.RuleResponse
+	ruleName := ruleResponses[0].Name()
+	ruleType := ruleResponses[0].RuleType()
+	ruleMesssage := ""
+	for _, rsp := range ruleResponses {
+		if rsp.Status() == engineapi.RuleStatusFail {
+			failRuleResponses = append(failRuleResponses, rsp)
+		} else if rsp.Status() == engineapi.RuleStatusError {
+			errorRuleResponses = append(errorRuleResponses, rsp)
+		} else if rsp.Status() == engineapi.RuleStatusPass {
+			passRuleResponses = append(passRuleResponses, rsp)
+		} else if rsp.Status() == engineapi.RuleStatusSkip {
+			skipRuleResponses = append(skipRuleResponses, rsp)
+		}
+	}
+	var combineRuleResponses []engineapi.RuleResponse
+	if len(errorRuleResponses) > 0 {
+		for _, errRsp := range errorRuleResponses {
+			ruleMesssage += errRsp.Message()
```

Comment:
It would be good to combine error messages with `;` to make it more readable.

Reply:
done